test/db/tools/rabin2

```diff
@@ -1044,3 +1044,220 @@ EXPECT=<<EOF
 1
 EOF
 RUN
+
+NAME=rabin2 -g
+FILE=bins/elf/analysis/x86-helloworld-clang
+CMDS=!rabin2 -g ${R2_FILE}
```

Comment:
this is making r2 to load the file twice. make `FILE=--` and then just put the file path in here